_layouts/search.html

```diff
@@ -43,6 +43,6 @@ <h5 class="modal-title" id="modalLabel">Lunr Search Options</h5>
 </div>
```

Comment:
Here are the search bar and the button not translated.

Also the display of how many results the search returns: 
![image](https://github.com/Stadt-Geschichte-Basel/forschung.stadtgeschichtebasel.ch/assets/73133626/997ba53e-8cbc-4eb5-b05b-72bee3585bbb)
But this comes probably directly from lunr.js?

Reply:
fixed it as well as "Enter your search term..." and Search button.